source/includes/facts-kubectl-plugin.rst

```diff
@@ -0,0 +1,20 @@
+.. default-domain:: minio
+
+.. start-kubectl-minio-requires-operator-desc
+
+.. admonition:: Command Requires MinIO Operator
+   :class: note
+
+   Use the following command to validate that the operator
+   is online and available prior to running this command:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl get pods -o wide | awk 'NR == 1 || /minio-operator/'
```

Comment:
this assumes the namespace is set to the same one as the minio operator. One alternative is `kubectl get deployments -A`  or `kubectl get deployments -A --field-selector=metadata.name=minio-operator`

Reply:
Fair point - I wonder if modifying to `kubectl get pods --all-namespaces -o wide` would work then, since that searches all possible namespaces. I'll try your options as well and see which one awks cleanly :)

Reply:
`kubectl get deployments` doesn't seem to return the `minio-operator` pod.

`kubectl get pods -o wide -A | awk "NR==1 || /minio-operator/'` does seem to work, so going with that.

Reply:
Deployments returns the deployment as minio operator is installed as a deployment. A deployment has pods. Not sure what the goal is here but kubectl cli has the capability to query and report without having to bring out awk and pipe. Maybe the filter option can be selected to see if the pod is from minio operator?

Reply:
oh - I somehow skipped right past the very obvious first line of the output. You're very correct, and your second example works. The goal was just an easy-ish way to see if the operator is already running.

Reply:
@kerneltime PR fixed.